scipy/sparse/linalg/_isolve/tests/test_iterative.py

```diff
@@ -530,6 +530,27 @@ def test_x0_equals_Mb(solver):
             assert_normclose(A.dot(x), b, tol=tol)
 
 
+@pytest.mark.parametrize(('solver', 'solverstring'), [(tfqmr, 'TFQMR')])
```

Comment:
I assume you're using `parametrize` here in order to extend this test to other solvers next? Otherwise `tfmqr` could simply be hardcoded instead.

Reply:
Dear Ralf @rgommers, thank you for your review!
> I assume you're using parametrize here in order to extend this test to other solvers next?

Yes, I will add Conjugate Rsidual (CR) implement later (PR: gh-15683)

Reply:
sounds good, thanks!